notebooks/pixiedust_facebook_analysis.ipynb

```diff
@@ -266,6 +266,14 @@
     "\n",
     "**Select the cell below and place your cursor on an empty line below the comment.** Load the CSV file you want to enrich by clicking on the 10/01 icon (upper right), then click `Insert to code` under the file you want to enrich, and choose `Insert Pandas DataFrame`.\n",
     "\n",
+    "### Note: There is an issue (https://github.com/IBM/pixiedust-facebook-analysis/issues/39)\n",
+    "\n",
```

Comment:
This is double-spaced in the notebook which looks kind of broken up and strange.